newsletter.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: guide
+title: Newsletter
+description: Keep up with all the activity in the Bitcoin Design Community.
+permalink: /newsletter/
+main_nav: true
+nav_order: 5
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/newsletter/newsletter-preview.jpg
+---
+
+{% include picture.html
+   image = "/assets/images/newsletter/newsletter.jpg"
+   retina = "/assets/images/newsletter/newsletter@2x.jpg"
+   mobile = "/assets/images/newsletter/newsletter-mobile.jpg"
+   mobileRetina = "/assets/images/newsletter/newsletter-mobile@2x.jpg"
+   alt-text = "Grid of images from the Bitcoin Design newsletter content"
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# The Bitcoin Design Newsletter
+
+Don't have the time to keep up with all the activity of bitcoin designers in and around Slack? Quickly catch up with [the newsletter](https://bitcoindesign.substack.com). Every 3 weeks, you get a concise summary of recent activity in your inbox.
+
+<iframe src="https://bitcoindesign.substack.com/embed" width="480" height="320" style="border:1px solid #EEE; background:white; width: 100%;" frameborder="0" scrolling="no"></iframe>
```

Comment:
Good one, forgot about that. Totally cool to remove it. It might even be the better choice to have a button instead that links out. Who signs up for a newsletter without checking it first anyways.

Reply:
Updated.